src/features/event_episodes/components/EpisodesTimeline/EpisodesTimeline.tsx

```diff
@@ -65,6 +65,7 @@ export function EpisodesTimeline({ episodes }: { episodes: Episode[] }) {
         stack={timelineState.settings.stack}
         cluster={timelineState.settings.cluster}
         onSelect={onSelect}
+        margin={{ axis: 2 }}
```

Comment:
As margin prop is static I would suggest to move it to constant object out of component, to have static link on it.
Here it would be created new object on every render.

Reply:
Agree. Actually, we don't have any rendering optimizations for `EpisodesTimeline` component, so I don't worry about renders here

Reply:
@propakov Fixed